.github/workflows/build.yml

```diff
@@ -0,0 +1,102 @@
+name: moonlight-android
+
+on: [push, pull_request]
+
+jobs:
+  build:
+    name: Build
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: android-actions/setup-android@v3
+      with:
+        cmdline-tools-version: 9477386
+
+    - uses: actions/setup-java@v4
+      with:
+        java-version: 21
+        distribution: 'adopt'
+
+    - name: Set up environment
+      run: |
+        echo "ANDROID_NDK_HOME=$ANDROID_NDK_HOME" >> $GITHUB_ENV
+        echo "ANDROID_SDK_ROOT=$ANDROID_HOME" >> $GITHUB_ENV
+        echo "COMMIT_PREFIX=$(echo ${{ github.sha }} | cut -c1-8)" >> $GITHUB_ENV
+
+    - name: Checkout moonlight-android repo
+      uses: actions/checkout@v4
+      with:
+        fetch-depth: 0
```

Comment:
What's the reason for fetching the full history?

Reply:
I have seen some issues in the past, that why I use this for pipelines, I have removed it.